modules/system/models/MailTemplate.php

```diff
@@ -117,38 +119,71 @@ public static function syncAll()
         }
     }
 
+    /**
+     * Fired after the model has been fetched.
+     * @return void
```

Comment:
We're aiming for an empty line between the method description and the rest of the docblock on every method.
```suggestion
     * Fired after the model has been fetched.
     *
     * @return void
```